traits/traits.py

```diff
@@ -86,7 +86,7 @@
 #-------------------------------------------------------------------------------
 
 PasswordEditor      = None
-MultilineTextEditor = None
+MultilineTextEditor = {}
```

Comment:
Can we rename this to `MultilineTextEditors`, now that it's a container for editors?